conf/deployment/baremetal/upi_1az_rhcos_multus_nvme_intel_3m_3w.yaml

```diff
@@ -13,6 +13,11 @@ ENV_DATA:
   is_multus_enabled: true
   multus_public_net_interface: 'enp1s0f1'
   multus_cluster_net_interface: 'enp1s0f1'
+  worker_node_network_configuration: [
+    ["ceph-public-net-shim-argo005","argo005.ceph.redhat.com","192.168.252.1","24","192.168.20.0/24"],
```

Comment:
I think we should move those names:
argo005","argo005.ceph.redhat.com"
to specific credential conf. Imagine someone else would like to run whit this BM configuration but on completely different env. So it will be confusing here and I think should be stored in specific credential conf (jenkins secret) for specific env.

@dahorak WDYT?

Reply:
@OdedViner as Petr suggested, this is specific for the _BM Octo Argo_ environment, so it should be configured in the respective "secrets" configuration file (`secrets/bm-octo-argo` in _ocs4-jenkins_ repo).

Also the configuration for BM environment looks like this (see also description for [PR#9274](https://github.com/red-hat-storage/ocs-ci/pull/9274)):
```
ENV_DATA:
  baremetal:
    ...
    servers:
      host001.example.com:
        <configuration-for-host001>
      host002.example.com:
        <configuration-for-host001>
      ...
```
which means, that each server already have its own section in the configuration file where is (beside other thinks) configured the role of the server, network configuration, etc. So I think those new parameters could be added to the configuration of each respective server, WDYT?

Reply:
@dahorak @petr-balogh  done 
Opened a MR https://url.corp.redhat.com/e2f27e3